### DISCUSSION TITLE
git status On branch master nothing to commit, working tree clean

### DISCUSSION THREAD

**Discussion by rawat09** [community/community#37371] [Other Feature Feedback, Questions, & Ideas]
git status
On branch master
nothing to commit, working tree clean

**Comment by airtower-luna** [comment]
I assume if you're posting here you have some kind of a problem, but what you posted isn't an error message. To get help, describe your actual problem. A good problem description should include at least:

* What exactly are you doing?
* What happens?
* What would you expect to happen instead?

Try to be as precise as possible.
